applications/socbayio.md

```diff
@@ -0,0 +1,112 @@
+
+
+* Are there any other projects similar to yours in the IPFS/Crust ecosystem? 
+
+Lbry has their own video sharing platform https://odysee.com/ (previously lbry.tv) It's not fully distributed because the contents are guaranteed by private Lbry nodes. Public nodes don't have incentive to keep their nodes alive. Odysee has more than 15M visits each month. People start using distributed video sharing platform because they care more and more about the cencorship.
+
+D.tube is a distributed video sharing blockchain, this project is awesome but the weakest point of the project is that people should be responsable for the storage of their videos. This project could work for a certain amount of people but for public audience it doesn't work. I made a survey with my friends who are engineers in IT fields, they are not aware about distributed storage protocols and distributed storage platforms, so asking them to store videos themself without using centralized platforms is a diffucult mission.
+
+
+  * If so, how is your project different?
+  
+Socbay.io was built by Crust Network supporters therefore it's for Crust Network first. Socbay.io is fully decentralized and is maintained by donation and advertisement. 
+  
+
+## Team
+
+### Team members
+* Nguyen Trong Nghia
+
+### Contact
+* **Contact Name:** Nguyen Trong Nghia
+* **Contact Email:** trongnghia.nguyen@outlook.com
+
+### Legal Structure 
+* **No legal structure** 
+
+### Team Code Repos
+* https://github.com/nghianguyen119/
+
+### Team LinkedIn Profiles
+* https://www.linkedin.com/in/trongnghia/
+
+
+### Overview
+* **Total Estimated Duration:** 4 months
+* **Full-Time Equivalent (FTE):**  1
+* **Total Costs:** 15000 USD
+
+### Milestone 1: MVP website
+* **Estimated Duration:** 2 months
+* **FTE:**  1
+* **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. |MVP website|Implement basic functions for the website| 
+| 2. |Self uploading video|Implement private node to store temporary videos, user must place the order themself on Crust Apps|  
+
+
+
+### Milestone 2: Stable website
+
+* **Estimated Duration:** 2 months
+* **FTE:**  1
+* **Costs:** 5,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. |Private gateway|Create private gateway|
+| 2. |Additional features|Implement additional functions for the website (e.g. chat)| 
+| 3. |Website Security|Strengthen the security to avoid malicious attacks|
+| 4. |Free uploading video|Using donation to create a common budget where users could use to store their video easily with Socbay.io|  
```

Comment:
Design the CRU payment system. Make it easier for users to use and highlight the concept of decentralized video sites.